.github/workflows/build-prover-fri-gpu-gar.yml

```diff
@@ -44,3 +44,13 @@ jobs:
           push: true
           tags: |
             us-docker.pkg.dev/matterlabs-infra/matterlabs-docker/prover-fri-gpu-gar:2.0-${{ inputs.image_tag_suffix }}
+
+      - name: Login to asia-southeast1 GAR
```

Comment:
not Asia Southeast, but just asia

please also add pushing to Europe

Reply:
updated, and added for Europe too.